sft_config/agent-application-config.tpl

```diff
@@ -81,3 +81,15 @@ sender:
       filenameRegex: .*
       maxThreadPoolSize: 3
       threadPoolSize: 3
+
+    - name: internal/DA/inbound/SendTo-CRE
```

Comment:
I'm not sure this lines up with the - name above.  Can you fix it?   I have seen issues when things aren't perfect in the past